network-api/networkapi/mozfest/templates/partials/signup.html

```diff
@@ -11,8 +11,8 @@
           class="join-us mb-5"
           data-form-position="newsletter_signup_sidebar"
           data-signup-id="{{ page.signup.id|unlocalize }}"
-          data-cta-header="{{ page.signup.header | escape }}"
-          data-cta-description="{{ page.signup.description | escape }}"
+          data-cta-header="{{ page.signup.localized.header | escape }}"
```

Comment:
Using https://github.com/mozilla/foundation.mozilla.org/pull/7658/files for reference, to make sure that the signup snippets are still localized.

Reply:
Have you considered checking how many extra database queries it adds when we reuse the `.localized` method multiple times? I wonder if it would make sense to pull the localized signup into a variable and access the properties on that.